### PR TITLE
[datadog_service_level_objectives] Update docs to clarify primary time frame fields

### PR DESCRIPTION
**Pull request by nkzou**
Docs are not clear about which field is used to specify the primary time frame.

### CODE REVIEW

File: datadog/resource_datadog_service_level_objective.go [hunk 1]
```diff
@@ -94,21 +94,21 @@ func resourceDatadogServiceLevelObjective() *schema.Resource {
...
-				Description:      "The objective's target in `(0,100)`.",
+				Description:      "The objective's target in `(0,100)`. This must match the corresponding thresholds of the primary time frame ",
 				Type:             schema.TypeFloat,
 				Optional:         true,
 				Computed:         true,
 				DiffSuppressFunc: suppressDataDogFloatIntDiff,
 			},
 			"warning_threshold": {
-				Description:      "The objective's warning value in `(0,100)`. This must be greater than the target value.",
+				Description:      "The objective's warning value in `(0,100)`. This must be greater than the target value, and match the corresponding thresholds of the primary time frame.",
```
Comment (maycmlee) [inline datadog/resource_datadog_service_level_objective.go:104]:
```suggestion
				Description:      "The objective's warning value in `(0,100)`. This must be greater than the target value and match the corresponding thresholds of the primary time frame.",
```

File: datadog/resource_datadog_service_level_objective.go [hunk 2]
```diff
@@ -122,7 +122,7 @@ func resourceDatadogServiceLevelObjective() *schema.Resource {
 				ValidateDiagFunc: validators.ValidateEnumValue(datadogV1.NewSLOTypeFromValue),
 			},
 			"force_delete": {
-				Description: "A boolean indicating whether this monitor can be deleted even if it’s referenced by other resources (e.g. dashboards).",
+				Description: "A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (e.g. dashboards).",
```
Comment (maycmlee) [inline datadog/resource_datadog_service_level_objective.go:125]:
```suggestion
				Description: "A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (for example, dashboards).",
```

File: docs/resources/service_level_objective.md [hunk 3]
```diff
@@ -84,15 +84,15 @@ resource "datadog_service_level_objective" "bar" {
 ### Optional
 
 - `description` (String) A description of this service level objective.
-- `force_delete` (Boolean) A boolean indicating whether this monitor can be deleted even if it’s referenced by other resources (e.g. dashboards).
+- `force_delete` (Boolean) A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (e.g. dashboards).
```
Comment (maycmlee) [inline docs/resources/service_level_objective.md:87]:
```suggestion
- `force_delete` (Boolean) A boolean indicating whether this monitor can be deleted even if it's referenced by other resources (for example, dashboards).
```

File: docs/resources/service_level_objective.md [hunk 4]
```diff
@@ -84,15 +84,15 @@ resource "datadog_service_level_objective" "bar" {
...
 - `groups` (Set of String) A static set of groups to filter monitor-based SLOs
 - `monitor_ids` (Set of Number) A static set of monitor IDs to use as part of the SLO
 - `query` (Block List, Max: 1) The metric query of good / total events (see [below for nested schema](#nestedblock--query))
 - `tags` (Set of String) A list of tags to associate with your service level objective. This can help you categorize and filter service level objectives in the service level objectives page of the UI. Note: it's not currently possible to filter by these tags when querying via the API
-- `target_threshold` (Number) The objective's target in `(0,100)`.
-- `timeframe` (String) The time frame for the objective. The mapping from these types to the types found in the Datadog Web UI can be found in the Datadog API documentation page. Valid values are `7d`, `30d`, `90d`, `custom`.
+- `target_threshold` (Number) The objective's target in `(0,100)`. This must match the corresponding thresholds of the primary time frame
```
Comment (maycmlee) [inline docs/resources/service_level_objective.md:92]:
```suggestion
- `target_threshold` (Number) The objective's target in `(0,100)`. This must match the corresponding thresholds of the primary time frame.
```

**Review by maycmlee — COMMENTED** [review]
Left some suggestions

File: docs/resources/service_level_objective.md (hunk 4 above)
Comment (nkzou) [reply docs/resources/service_level_objective.md]: Fixed in [f421d46](https://github.com/DataDog/terraform-provider-datadog/pull/1774/commits/f421d461db0f71dfc3420048e70cf560ee979b8d)

File: datadog/resource_datadog_service_level_objective.go (hunk 2 above)
Comment (nkzou) [reply datadog/resource_datadog_service_level_objective.go]: Fixed in [f421d46](https://github.com/DataDog/terraform-provider-datadog/pull/1774/commits/f421d461db0f71dfc3420048e70cf560ee979b8d)

File: datadog/resource_datadog_service_level_objective.go (hunk 1 above)
Comment (nkzou) [reply datadog/resource_datadog_service_level_objective.go]: Fixed in [f421d46](https://github.com/DataDog/terraform-provider-datadog/pull/1774/commits/f421d461db0f71dfc3420048e70cf560ee979b8d)

**Review by zhengshizhao — APPROVED** [review]

**Review by skarimo — APPROVED** [review]

**Review by ddjamesfrullo — APPROVED** [review]

**Pull request merged**